### PR TITLE
Add acronyms

### PR DESCRIPTION
**Pull request by ravaan**
Some acronyms I thought were missing

CMV: Change My View
DAE: Does Anyone Else
ELI5: Explain Like I'm 5 (years old)
LSHMSFOAIDMT: Laughing So Hard My Sombrero Falls Off and I Drop My Taco
NSFL: Not Safe For Life (gory or gross)
YSK: You Should Know

### CODE REVIEW

**Comment by mkfeuhrer** [comment]
@ravaan 

- Can you remove `LSHMSFOAIDMT` - This is too big to be an acronym.
- Remove the content in brackets for NSFL.

Thanks!

**Comment by ravaan** [comment]
Done.

IMO - LSHMSFOAIDMT is a hilarious one that should be there in the collection.
